src/logout/GeneralLogoutHandler.ts

```diff
@@ -28,6 +28,24 @@ import ILogoutHandler from "./ILogoutHandler";
 import { inject, injectable } from "tsyringe";
 import { IStorageUtility } from "../storage/StorageUtility";
 
+/**
+ * This function removes all session-related information from storage.
+ * @param userId the session identifier
+ * @param storage the storage where session info is stored
+ * @hidden
+ */
+export async function clearSession(
```

Comment:
Should this function be in the `util/` folder (i.e. it's not really Logout-specific at all really)...?

Reply:
I'm not a big fan of a 'catch-all' `util` module. If clearing the session state is shared in the `util` module, then I see your point about just using `clearState` at the beginning of `login`, but given that a big refactoring is planned, I'd rather share high-level logic (e.g. logout at the beginning of login) rather than more specific implementation (e.g. clearing some specific  information from storage at login and logout).

Reply:
Sure, I agree (about having a catch-all `/util` module). I just meant at a high-level, conceptually. I guess I really meant moved to the `sessionInfo` folder, perhaps as a method on the `SessionInfoManager` class in fact...?